### PR TITLE
[Patch] Fix sim card selection for ufi-001c

### PR DESCRIPTION
**Pull request by 185264646**
@TravMurav Would you please test if this patch breaks the modem of uf896?

### CODE REVIEW

**Comment by TravMurav** [comment]
This seems to not cause any regressions on uf896, thanks!

I think it's better to squash these two commits into a single one, saying it fixes a mistake in the original series and add a Fixes: tag pointing to your commit adding the dts

**Comment by 185264646** [comment]
> This seems to not cause any regressions on uf896, thanks!
> 
> I think it's better to squash these two commits into a single one, saying it fixes a mistake in the original series and add a Fixes: tag pointing to your commit adding the dts

Thanks for your comment. I had pushed my latest commit.

File: arch/arm64/boot/dts/qcom/msm8916-thwc-ufi001c.dts [hunk 1]
```diff
@@ -34,6 +39,25 @@
 	pins = "gpio20", "gpio21", "gpio22";
 };
 
-&sim_ctrl_default {
-	pins = "gpio1", "gpio2";
+/* This selects the external SIM card slot by default */
+&msmgpio {
+	sim_ctrl_default: sim-ctrl-default-state {
+		esim-sel {
```
Comment (TravMurav) [inline arch/arm64/boot/dts/qcom/msm8916-thwc-ufi001c.dts]: I think the schema requires `-pins` suffix for those subnodes

https://elixir.bootlin.com/linux/v6.2/source/Documentation/devicetree/bindings/pinctrl/qcom,msm8916-pinctrl.yaml#L43

Comment (185264646) [reply arch/arm64/boot/dts/qcom/msm8916-thwc-ufi001c.dts]: > I think the schema requires `-pins` suffix for those subnodes
> 
> https://elixir.bootlin.com/linux/v6.2/source/Documentation/devicetree/bindings/pinctrl/qcom,msm8916-pinctrl.yaml#L43

fixed

File: arch/arm64/boot/dts/qcom/msm8916-thwc-ufi001c.dts [hunk 2]
```diff
@@ -34,6 +39,25 @@
...
+			pins = "gpio2";
+			bias-disable;
+			output-high;
+		};
+
+		sim-en-pins {
+			pins = "gpio1";
+			bias-disable;
+			output-low;
+		};
```
Comment (wonderfulShrineMaidenOfParadise) [inline arch/arm64/boot/dts/qcom/msm8916-thwc-ufi001c.dts]:
```suggestion
		sim-en-pins {
			pins = "gpio1";
			bias-disable;
			output-low;
		};

		sim-sel-pins {
			pins = "gpio2";
			bias-disable;
			output-high;
		};
```

**Comment by 185264646** [comment]
I suppose this pr can be closed now?

**Comment by TravMurav** [comment]
Thanks! Applied upstream patch to the wip/msm8916/6.2

**Pull request closed without merging**